vcr/issuer/leiaStore.go

```diff
@@ -0,0 +1,82 @@
+package issuer
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	ssi "github.com/nuts-foundation/go-did"
+	"github.com/nuts-foundation/go-did/did"
+	"github.com/nuts-foundation/go-did/vc"
+	"github.com/nuts-foundation/go-leia/v2"
+	"github.com/nuts-foundation/nuts-node/vcr/credential"
+	"path"
+)
+
+// leiaStore implements the issuer Store interface. It is a simple and fast json store.
+// Note: It can not be used in a clustered setup.
+type leiaStore struct {
+	collection leia.Collection
+}
+
+// NewLeiaStore creates a new instance of leiaStore which implements the Store interface.
+func NewLeiaStore(dataDir string) (Store, error) {
+	dbPath := path.Join(dataDir, "vcr", "issued-credentials.db")
+	store, err := leia.NewStore(dbPath, false)
+	if err != nil {
+		return nil, fmt.Errorf("failed to create leiaStore: %w", err)
+	}
+	collection := store.Collection("issuedCredentials")
+	newLeiaStore := &leiaStore{collection: collection}
+	if err := newLeiaStore.createIndices(); err != nil {
```

Comment:
must this always be called?

Reply:
Cannot think about a use-case we want a store without it indices. Perhaps in a test condition, but the current tests do not demand it. We can always add a param later.